tests/integration/helpers.py

```diff
@@ -393,6 +393,13 @@ async def deploy_and_relate_bundle_with_postgresql(
             ]
 
             # Remove PostgreSQL and relations with it from the bundle.yaml file.
+            config = data["applications"]["postgresql"]["options"]
+            if config.get("experimental_max_connections", 0) > 200:
+                config["experimental_max_connections"] = 200
```

Comment:
Landscape sets it to 500, limiting to 200

Reply:
OK, as it is a test only in limited HW env (CI runner).